### ISSUE TITLE
Event Callback Error

### ISSUE THREAD

**Issue by Noob1e** [Ostoic/RaidBrowser#20]
```
event callback error:
 Interface\AddOns\RaidBrowser\core.lua:768: invalid pattern capture
```

**Comment by Ostoic** [comment]
I have a few ideas for how to bypass this for now (wrap pcall the raid_info function). I'll have to add some better error messages to figure out which pattern is causing the issue, though. I'll see if I can figure this out tonight when I can test it out. 
Thanks for the issue!

**Comment by Ostoic** [comment]
Btw, does the addon stop working after this happens? Is it just an error message that's popping up in chat occasionally? I believe I attempted to prevent addon crashes in the latest release, so I'm curious if it worked. I can't seem to reliably reproduce the bug, so I wasn't able to fix it the last time I tried to.

**Comment by Noob1e** [comment]
Nope, errors don't stop the addon anymore. But for a long time I didn't get any other error messages from this addon so I just wanted to share it.
